run_ddt_construct.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Running the original cut-based selection (No RT DDT)
+rm ./models/cutbased_ddt_map_ANv6.json
+python apply_DDT.py --analysis_type cut-based --rt_sel 1.18 --ddt_map_file ./models/cutbased_ddt_map_ANv6.json --bkg_files "root://cmseos.fnal.gov//store/user/lpcdarkqcd/boosted/skims_20241030_hadd/Summer20UL*/*.npz" --sig_files "root://cmseos.fnal.gov//store/user/lpcdarkqcd/boosted/skims_20241030_hadd/Private*/*.npz" --plot 2D_DDT_map fom_significance
```

Comment:
preferably, repeated values like `"root://cmseos.fnal.gov//store/user/lpcdarkqcd/boosted/skims_20241030_hadd/Summer20UL*/*.npz"` should be encoded into variable such that the script can be updated consistently with as few edits as possible